README.md

```diff
@@ -63,9 +63,10 @@ Fetching data
 
 Ensure that the virtualenv is activated, and then run:
 
-    GAAUTH='...' PYTHONPATH=. python scripts/fetch.py page-traffic.dump
+    GAAUTH='...' PYTHONPATH=. python scripts/fetch.py page-traffic.dump 14
 
-(Where `GAAUTH` is the values obtained from the `setup_auth.py` script.)
+(Where `GAAUTH` is the values obtained from the `setup_auth.py` script and
```

Comment:
should that read "value" instead of "values"?

Reply:
Yes it should. Fixed. 👍